module/input/SensorFilter/src/SensorFilter.hpp

```diff
@@ -97,7 +95,6 @@ namespace module::input {
             FootDownMethod(std::string const& str) {
                 // clang-format off
                         if      (str == "Z_HEIGHT") { value = Value::Z_HEIGHT; }
-                        else if (str == "LOAD") { value = Value::LOAD; }
```

Comment:
Surely we can cut this down more, there's like over 60 lines just to define foot down :laughing: